tools/fuzz/fuzz.cpp

```diff
@@ -215,23 +219,11 @@ FuzzStatus ParseFlags(int argc, const char** argv, std::string* in_binary_file,
     return {FuzzActions::STOP, 1};
   }
 
```

Comment:
By removing the following, it looks like we make it possible to pass an input file that does not end in .spv.  Am I correct, and if so is it intentional?  The help for the tool says that the input file must be a .spv file.

Reply:
Yes it is intentional. I'd like to pass a file with extension ".spv_orig". I will update the help text.